tv_show/src/components/features/shows/ReviewForm/ReviewForm.tsx

```diff
@@ -0,0 +1,62 @@
+import { CustomRatingInput } from '@/components/shared/input/CustomRatingInput/CustomRatingInput';
+import { IReview } from '@/typings/review';
+import { Flex, Heading, Input, Button } from '@chakra-ui/react';
+import { useState } from 'react';
+
+interface ReviewFormProps {
+	addShowReview: (review: IReview) => void;
+}
+
+export const ReviewForm = ({ addShowReview }: ReviewFormProps) => {
+	const [internalValue, setInternalValue] = useState(0);
+  const [commentValue, setCommentvalue] = useState("");
+
+	const onClickHandler = () => {
+		const commentInput = commentValue;
+		const ratingInput = internalValue;
+
+		if (!commentInput || !ratingInput) {
+			return;
+		}
+
+		const newReview: IReview = {
+			comment: commentInput,
+			rating: ratingInput,
+			email: 'test@gmail.com',
+			avatar: '',
+		};
+
+		addShowReview(newReview);
+	};
+
+	const onChange = (index: number) => {
+		setInternalValue(index);
+	};
+
+  const onChangeComment = (value: string) => {
+    setCommentvalue(value);
+  }
+
+	return (
+		<Flex direction="column" gap={4} width="60%">
```

Comment:
S obzirom da se komponenta zaove ReviewForm, ocekivao bih da je ovdje mozda form element.
Bolji pristup bi bio:

`<Flex direction="column" as="column" gap={4} width="60%" onSubmit={onClickHandler}>`

i onda dolje u button elementu maknuti onClick i postaviti `type="submit"`

Ovo je način na koji ćemo ubuduće raditi forme :)